src/aotextarea.cpp

```diff
@@ -32,9 +32,8 @@ void AOTextArea::append_chatmessage(QString p_name, QString p_message,
     p_message += " ";
   }
 
-  QString result = p_message.toHtmlEscaped()
-                       .replace("\n", "<br>")
-                       .replace(url_parser_regex, "<a href='\\1'>\\1</a>");
+  QString result = QString::fromStdString(this->closetags(p_message.replace("\n", "<br>").toStdString()));
```

Comment:
funny conversion to std:: moment, doesn't Qt have its own version of regex?
https://doc.qt.io/qt-5/qregexp.html#details